servicex/values.yaml

```diff
@@ -112,6 +112,9 @@ transformer:
     minReplicas: 1
     maxReplicas: 20
   pullPolicy: Always
+  # Add priorityClassName to control priority and preemption of the transformer
+  # pods, Priorityclass needs to be created in the cluster  
+  # priorityClassName: 
```

Comment:
Actually, this works if you just leave a blank after the colon:

```yaml
  pullPolicy: Always
  priorityClassName:
```
It would be a little cleaner that way instead of commented out lines.